app/src/main/java/com/github/libretube/api/obj/Comment.kt

```diff
@@ -6,7 +6,7 @@ import kotlinx.serialization.Serializable
 data class Comment(
     val author: String,
     val commentId: String,
-    val commentText: String,
+    val commentText: String?,
```

Comment:
Server was returning null for some comments and kotlinx-serializer could not succeed.